### ISSUE TITLE
Arg normalize to sklearn.linear_model.LinearRegression has been removed in version 1.2

### ISSUE THREAD

**Issue by carlsmedstad** [rasbt/mlxtend#1035]
#### Describe the bug

The following will fail for scikit-learn version 1.2:

```python
sklearn.linear_model.LinearRegression(normalize=True)
```

Docs for 1.0: https://scikit-learn.org/1.0/modules/generated/sklearn.linear_model.LinearRegression.html?highlight=linearregression#sklearn.linear_model.LinearRegression

Docs for 1.2: https://scikit-learn.org/1.2/modules/generated/sklearn.linear_model.LinearRegression.html?highlight=linearregression#sklearn.linear_model.LinearRegression

**Comment by carlsmedstad** [comment]
After these changes the test pass:

```sh
sed -i 's/LinearRegression(normalize=True)/LinearRegression()/g' \
  mlxtend/regressor/tests/test_stacking_regression.py \
  mlxtend/regressor/tests/test_stacking_cv_regression.py
```

**Comment by namanmistry** [comment]
Hey, In the old docs of sklearn for `normalize` parameter it was written: 
```
This parameter is ignored when fit_intercept is set to False. If True, the regressors X will be normalized before regression by subtracting the mean and dividing by the l2-norm. 

```

And `fit_intercept` defaults to `True` in the newer version of sklearn. So can we directly remove `normalize` argument? I think it wont effect the functionality because it is still normalizing.

**Comment by rasbt** [comment]
Thanks for the note & thanks for addressing that in PR #1036!
